GlobalConstantsService/data/globalConstants_01.txt

```diff
@@ -40,6 +42,8 @@ double physicsParams.Cu.decayFraction  = 0.07;
 double physicsParams.Se.decayFraction  = 0.07;
 double physicsParams.Sb.decayFraction  = 0.04;
 double physicsParams.Au.decayFraction  = 0.03;
+double physicsParams.C.decayFraction  = 0.86;
+double physicsParams.IPA.decayFraction  = 1.0; 
```

Comment:
The other entries here are physics numbers, the 1.0 is what you want for a particular application.  Please change this to the physical number, it will just reduce the efficiency of the Michel electron calibration case.

Reply:
Yes, I mean we could just use Carbon if that's the case and remove thje "IPA" instance

Reply:
I guess, maybe not, because at some point we will update this at some point so maybe leave the IPA for now and add the TODO